### PR TITLE
Add changelog for adding benchmark input

### PR DESCRIPTION
**Pull request by leehinman**
## What does this PR do?

Add changelog entry for adding benchmark input

## Why is it important?

Document adding enhancement

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/elastic-agent#changelog)
- [ ] I have added an integration test or an E2E test

## Related issues

-  Relates elastic/beats#39789

## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...

### CODE REVIEW

File: changelog/fragments/1717523676-Add-benchmark-input.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Change summary; a 80ish characters long description of the change.
+summary: Add benchmark input
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+# NOTE: This field will be rendered only for breaking-change and known-issue kinds at the moment.
+#description:
+
+# Affected component; a word indicating the component this changeset affects.
+component: agent
```
Comment (ycombinator) [inline changelog/fragments/1717523676-Add-benchmark-input.yaml:22]: I don't think this field gets used anywhere but, just for consistency with [other changelog entries](https://github.com/elastic/elastic-agent/tree/main/changelog/fragments), let's use `elastic-agent` here.
```suggestion
component: elastic-agent
```

Comment (leehinman) [reply changelog/fragments/1717523676-Add-benchmark-input.yaml]: switched.  Thanks

**Review by ycombinator — APPROVED** [review]
Thanks for adding this entry!

**Comment by jlind23** [comment]
Everything is green but SonarQube is stuck in an `Expected` state hence force merging.

**Pull request merged**